docs/admin/install/serving/install-serving-with-yaml.md

```diff
@@ -319,8 +324,8 @@ The tabs below expand to show instructions for installing each Serving extension
 === "HPA autoscaling"
 
     Knative also supports the use of the Kubernetes Horizontal Pod Autoscaler (HPA)
-    for driving autoscaling decisions. The following command will install the
-    components needed to support HPA-class autoscaling:
+    for driving autoscaling decisions. Install the components needed to support HPA-class
```

Comment:
As for below, I'd keep any lines that call out an action e.g. "Install..." on a new line with a line break between this and the paragraph to make it easier to see the step

Reply:
I also noticed this. It's part of my suggestions below if you want to apply this change.